.github/ISSUE_TEMPLATE/proposed-feature-addition.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Proposed Feature/Addition
+about: Suggest an addition to Parsons
+title: "[Feature/Addition]"
+labels: enhancement
+assignees: ''
+
+---
+
```

Comment:
Similar to the bug report template, can we add a priority prompt? Such as:

Please indicate whether fixing this bug is high, medium, or low priority for you. If the issue is time-sensitive for you, please let us know when you need it addressed by.